### ISSUE TITLE
AdminLTE 2.1.0-beta

### ISSUE THREAD

**Issue by almasaeed2010** [almasaeed2010/AdminLTE#345]
# AdminLTE 2.1.0 is almost ready
Final testing should be done soon and a production release will be out in a few days. Please take some time testing the beta version and report any problems here. The live demo for the beta release can be found here:
https://almsaeedstudio.com/themes/AdminLTE-dev/index2.html

![screen shot 2015-04-16 at 11 50 41 am](https://cloud.githubusercontent.com/assets/1512664/7185107/dac44f04-e42e-11e4-931f-900eae9fd408.png)

**Comment by yufieko** [comment]
when activate Fixed Layout, sub menu in left sidebar (in mini sidebar mode) is not showing
![lte2](https://cloud.githubusercontent.com/assets/7820512/7193567/cfc025a8-e4cc-11e4-95ea-77ed88756dec.png)

**Comment by adaniello** [comment]
Seems only a issue about real-time previewing (javascript activation of feature) when you check the option: if you go in other pages with fixed layout, sub menu appears.

**Comment by kressly** [comment]
@almasaeed2010 
Wonderful Thanks a lot for the light skins :+1: 
Still checking and will get back to you.

**Comment by pratik60** [comment]
@almasaeed2010 Hey, the sidebar options don't expand on the fixed layout, when the sidebar is collapsed. On a non-fixed layout, if you hover over the icon in the collapsed sidebar, the menu appears on the side. But not so on a fixed layout....That's a critical bug, na?

**Comment by almasaeed2010** [comment]
@yufieko @pratik60 since slimscroll prevents sidebar mini from behaving properly, I added a new feature that enables the sidebar to expand on hover. If both sidebar mini and fixed layout are enabled the option for expand on hover gets enabled automatically. Otherwise, you can enable this option from app.js. The new fix will be available for preview on the [dev-demo](https://almsaeedstudio.com/themes/AdminLTE-dev/index2.html) shortly but the updates are available in master.

**Comment by pratik60** [comment]
this is my 

```
  <body class="logged-in skin-blue sidebar-mini fixed">
```

I am using the latest app.js from master, but the menu option aren't appearing when its collapsed?

The code while its collapsed

```
  <body class="logged-in skin-blue sidebar-mini fixed sidebar-collapse">
```

Any options which I had to change?

**Comment by almasaeed2010** [comment]
@pratik60 you shouldn't have to do anything else. Are you sure you are loading the newest app.js and css files? 

If you have the latest files, try enabling the ```expandOnHover``` option in app.js.

**Comment by pratik60** [comment]
Can't trace why it was happening, but seems to be fixed with the latest app.js. 

But I don't like the expand on hover tbh, I really felt rather than expand on hover, the way it happens in non fixed layout is really good. But I'm guessing you ran into difficulties and weren't able to implement that for a fixed layout.

**Comment by almasaeed2010** [comment]
@pratik60 you are right. I mentioned before that the fixed layout implements a technique to make the sidebar scrollable which is preventing horizontal overflow to be visible. I tried multiple suggested solutions. None of them, however, worked. Thus, I had to implement a different approach. I agree that it might not be the prettiest, but I do think it serves the purpose correctly. Thank you for reporting the issue.
